server/base/src/main/java/org/apache/accumulo/server/conf/ServerConfigurationFactory.java

```diff
@@ -129,7 +129,7 @@ public NamespaceConfiguration getNamespaceConfiguration(NamespaceId namespaceId)
       context.getPropStore().registerAsListener(NamespacePropKey.of(context, namespaceId),
           deleteWatcher);
       var conf = new NamespaceConfiguration(context, namespaceId, getSystemConfiguration());
-      ConfigCheckUtil.validate(conf);
+      ConfigCheckUtil.validate(conf, namespaceId.toString());
```

Comment:
Since these are strings, I think I would have preferred something like:

```java
ConfigCheckUtil.validate(conf, "Namespace id: " + namespaceId);
```

So we know it's a namespace and not a table ID. Same with the table IDs.